cmd/renterd/config.go

```diff
@@ -205,19 +205,17 @@ func setSeedPhrase() {
 // already set via environment variable or config file.
 func setAPIPassword() {
 	// retry until a valid API password is entered
-	for {
+	for len(cfg.HTTP.Password) < 4 {
 		fmt.Println("Please choose a password for the renterd admin UI.")
 		fmt.Println("This password will be required to access the admin UI in your web browser.")
 		fmt.Println("(The password must be at least 4 characters.)")
 
 		cfg.HTTP.Password = readPasswordInput("Enter password")
-		if len(cfg.HTTP.Password) >= 4 {
```

Comment:
Tbh I strongly prefer this. Duplicating the condition is a bit confusing.

Reply:
I did it to avoid an early return check but updated it.